students/Ruohan/lesson03/mailroom.py

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/env python3
+
+''' simple mailroom proram for a non profit'''
+
+import sys
+
+datas = [['William Gates III',  653784.39, 2],
+         ['Mark Zuckerberg', 16396.10, 3],
+         ['Jeff Bezos', 877.33, 1],
+         ['Paul Allen', 708.42, 3]]
+
+'''make a donors' name list'''
```

Comment:
Use `#` for a non-docstring comment. This is not a docstring comment, because it's neither for the module, nor for a function/class. Just `# make a donor's name list` should be fine.